addons/web/static/src/js/core/popover.js

```diff
@@ -26,37 +26,84 @@ odoo.define('web.Popover', function () {
             this.state = useState({
                 displayed: false,
             });
+
+            this._onClickDocument = this._onClickDocument.bind(this);
+            this._onScrollDocument = this._onScrollDocument.bind(this);
+            this._onResizeWindow = this._onResizeWindow.bind(this);
+
+            this._onScrollDocument = _.throttle(this._onScrollDocument, 50);
+            this._onResizeWindow = _.debounce(this._onResizeWindow, 250);
+
+            /**
+             * Those events are only necessary if the popover is currently open,
+             * so we decided for performance reasons to avoid binding them while
+             * it is closed. This allows to have many popover instantiated while
+             * keeping the count of global handlers low.
+             */
+            this._hasGlobalEventListeners = false;
+        }
+
+        mounted() {
+            this._compute();
```

Comment:
No call to super when you directly inherit from Component? I thought we agreed to always call super.

Reply:
We never use super in our refactoring so that must have been the original guideline. I have no idea now, please let us know if this changed. But as far as I can see, all components that have been merged so far in master don't call super either.

Reply:
@aab-odoo  I thought we agreed that it was not necessary on component methods?